### PR TITLE
test: add a script for multiplex pipeline stress test

### PR DESCRIPTION
**Pull request by keqiaozhang**
This script is used for simultaneous playback/capture test on
arbitrary multiple pipelines.

Signed-off-by: Zhang Keqiao <keqiao.zhang@linux.intel.com>

### CODE REVIEW

File: tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Simple script for simultaneous multiplex pipeline testing
+
+NUMBER=$#	# number of test pipeline
+PARAMS=$@	# name of test pipeline
+MAXLOOPS=1000	# loop number
+FORMATE=s16_le	# sample formate
+CHANNEL=2	# test channel number
+FREQENCY=48000	# sample frequency
+ITERVAL=5	# iterval time for each iteration
```
Comment (jajanusz) [inline tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh]: I guess you missed 'n' "iNterval" ;)
Also I think values can be smaller than 5 seconds, can it?

File: tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh [hunk 2]
```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+# Simple script for simultaneous multiplex pipeline testing
```
Comment (lgirdwood) [inline tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh]: license. BSD

File: tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+	echo "eg. ./multiple-pipeline-test.sh pcm1p pcm1c"
+	echo "it will check playback and capture on pcm1 at same time"
+	exit 1
+fi
+
+for i in $(seq 1 $MAXLOOPS)
+do
+	echo "Test: $i"
+	pipeline_test
+	sleep 1
```
Comment (lgirdwood) [inline tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh]: sleep duration should be passed in and can be < 1 second.

File: tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh (hunk 1 above)
Comment (keqiaozhang) [reply tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh]: Thanks for your review, indeed, this is a typo.

File: tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh (hunk 2 above)
Comment (keqiaozhang) [reply tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh]: Will update it with BSD license.

File: tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh [hunk 4]
```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Simple script for simultaneous multiplex pipeline testing
+
+# Copyright (c) 2016, Intel Corporation
```
Comment (lgirdwood) [inline tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh]: 2019 ?

File: tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+		if [ $? -ne 0 ];then
+			echo "Stop pipeline failure."
+			exit 1
+		fi
+	done
+}
+
+# run test
+if [ $# == 0 ]; then
+	echo "Usage:./multiplex-pipeline-test.sh '\$pipeline1' '\$pipeline2'.."
```
Comment (lgirdwood) [inline tools/test/qa/tests/stability_test/multiplex_pipeline-test.sh]: Probablu want to use $0 for script name here in case we ever rename it.

**Comment by lgirdwood** [comment]
@keqiaozhang can you update the changes incrementally as I would like to use this test myself today.

**Pull request merged**